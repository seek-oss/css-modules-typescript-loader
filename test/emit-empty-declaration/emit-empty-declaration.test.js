const fs = require('fs');
const compiler = require('../compiler.js');

test('Can emit valid declaration without classes', async () => {
  await compiler(require.resolve('./index.js'));
  
  const declaration = fs.readFileSync(
    require.resolve('./index.css.d.ts'),
    'utf-8'
  );

  expect(declaration).toMatchSnapshot();
});
