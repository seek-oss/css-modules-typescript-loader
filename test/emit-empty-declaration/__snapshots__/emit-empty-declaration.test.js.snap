// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Can emit valid declaration without classes 1`] = `
"// This file is automatically generated.
// Please do not change this file!
interface CssExports {

}
declare var cssExports: CssExports;
export = cssExports;
"
`;
